pkg/network/openshift_sdn.go

```diff
@@ -80,6 +88,18 @@ func renderOpenShiftSDN(conf *operv1.NetworkSpec, manifestDir string) ([]*uns.Un
 	}
 
 	objs = append(objs, manifests...)
+
+	klog.V(2).Infof("Openshift-SDN: Bootstrap Platform found %v  Azure platform value: %v", bootstrapResult.SDN.Platform, configv1.AzurePlatformType)
+	// Add Azure specific yaml needed to work around Azure specific issues
+	if bootstrapResult.SDN.Platform == configv1.AzurePlatformType {
+		manifestsAzure, err := render.RenderDir(filepath.Join(manifestDir, "network/openshift-sdn-azure"), &data)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to render azure specific manifests")
+		}
+		klog.V(2).Infof("Openshift-SDN: Starting Azure Append")
+		objs = append(objs, manifestsAzure...)
+		klog.V(2).Infof("Openshift-SDN: Azure Append Completed")
```

Comment:
This is like logging a malloc—seems redundant.  Probably just one log message (if even that) would suffice: 
```suggestion
		klog.V(2).Infof("Openshift-SDN: Including Azure-specific manifests")
		objs = append(objs, manifestsAzure...)
```